src/preprocess.py

```diff
@@ -411,9 +411,23 @@ def _get_task(name, args, data_path, scratch_path):
             tokenizer_name=args.tokenizer,
             **task_kw,
         )
+        task.load_data()
         utils.maybe_make_dir(os.path.dirname(pkl_path))
         pkl.dump(task, open(pkl_path, "wb"))
-    # task.truncate(max_seq_len, SOS_TOK, EOS_TOK)
+
+    return task
+
+
+def get_skeletal_task(task_name, args):
```

Comment:
I would rename this to something like prefetch_tasks_without_data, or make it more explicit what it's doing